### DISCUSSION TITLE
Assets (textures/models) location

### DISCUSSION THREAD

**Discussion by sw5ciprl** [efroemling/ballistica#378] [Q&A] [answered]
Hello, I'd like to make a contribution to in-game assets (textures/models), but can't seem to find where they are. Are they in this repo, or should I look somewhere else?

**Comment by Dliwk** [comment, marked as answer]
Hi! Textures/models and some parts of game binaries are stored externally on dedicated server and downloaded automatically. They are proprietary, but if you really want to contribute to them, you can directly contact Eric <support@froemling.net> (though he's unavailable right now).
